pages/components/image-input/responsive-images.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/4e7b8dc12c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../../../styles/styles.css">
+
+    <link rel="stylesheet" href="./responsive-images.css">
+    <link rel="stylesheet" href="../avatar/avatar.css">
+</head>
+<body>
+    <!-- navbar -->
+   
+        <nav class="navbar flex-sb">
+            <ul class="flex-row">
+                <li><img src="../../../assets/logo.jpg" alt="logo" class="mg-l-xs"></li>
+                <li><a href="../../../index.html" class="padding-m">Home</a></li>
+                <li><a href="../avatar/avatar.html" class="padding-m">Docs</a></li>
+            </ul>
+            <ul class="social-btns flex-row">
+                <li><a href="#" class="padding-m">Linkedin</a></li>
+                <li><a href="#" class="padding-m">Gitthub</a></li>
+            </ul>
+        </nav>
+
+        <div class="wrapper">
+            <ul class="sidebar flex-column-sidebar">
+                
+                <a href="../avatar/avatar.html" class="padding-m">Avatar</a>
+                <a href="../alert/alert.html" class="padding-m">Alert</a>
+                <a href="../badge/badge.html" class="padding-m">Badge</a>
+                <a href="../button/button.html" class="padding-m">Button</a>
+                <a href="../card/cards.html" class="padding-m">Card</a>
+                <a href="./responsive-images.html" class="padding-m">Image</a>
+                <a href="./input.html" class="padding-m">Input</a>
+                <a href="list.html" class="padding-m">List</a>
+                <a href="modal.html" class="padding-m">Modal</a>
+                <a href="other-utils.html" class="padding-m">Other Utilities</a>
+            </ul>
+      
+        
+            <div class="page-wrapper">
+                <h1>Responsive Image</h1>
+                <p class="mg-t-s">Responsive Image Components are used to display Images on websites, which are screen responsive. 
+                    They expand upto the width of the container they are in.</p>
+                <!-- the vertical card -->
+                <div>
+                    <h3 class="mg-t-s">Responsive Card</h3>
+                    <div class="flex-center mg-t-s mg-m">
+                        <div class="image-container">
+                            <img src="../../../assets/res-img.jpg" alt="responsive-image" class="responsive-img">
```

Comment:
You have used responsive img class,but it is not available in the image.css

Reply:
responsive-img is part of card styles, hence not included here.